docs/arch/adr-20.md

```diff
@@ -0,0 +1,47 @@
+# ADR 20: Transcription Task
+
+Created: November 12, 2019
+
+## Context
+
+IMLS funded us to experiment with transcription tools. The project [Anti-Slavery Manuscripts](https://www.antislaverymanuscripts.org/classify) has a workflow where volunteers transcribe lines of text first by marking the line, then adding the transcription in a sub-task. This task is slightly unique in that:
+
+- Incomplete classifications can be submitted
+- Previous transcriptions from caesar are loaded and presented as an option in the sub-task
+  - volunteers can select the previous transcription, edit it, and submit a new transcription
+  - lines with previous transcriptions or retired lines are displayed visually by color
+
+## Decision
+
+We will be porting the ASM functionality to the main classifier as a new transcription task. The task will be composite of:
+
+- A drawing line task for transcription that is created by two pointer down and up events to mark the points of the line.
+  - the starting point and ending point will have visual indicators communicating the direction of creation.
+  - the line color will indicate the current status of completion
+  - completed lines cannot be edited
+  - Note that ASM leveraged a hacked polygon tool to do this, but we should have a line tool variant specifically for this instead. Downstream aggregation can leverage polygon type aggregations if it fits.
+- A sub-task will display once the transcription line mark is made
+  - a text input will display suggestions from previous caesar aggregations.
+    - The suggestions can be selected and inserted as an editable value in the text input
+    - Grommet's [TextInput](https://storybook.grommet.io/?path=/story/textinput--suggestions) with the suggestions prop will be utilized so that it can be styleable with our theme.
+  - sub-task will not be a modal, but a movable div linked to the currently selected line mark
```

Comment:
The sub-task should be modal to the extent that creating new marks is blocked intil the current line is complete.

Reply:
The survey task opens its Identify modal inside the task area. Can we copy that behaviour for the transcription text task?

If the sub-task must be a popup, then it must not obscure the region of the page being transcribed and it must be moveable from the keyboard.

Reply:
The design of the sub-task is a movable pop-up. I'll be opening an issue to track this. If you'd like to suggest an alternative design, it can be discussed there and this ADR can be updated as needed.